ha/ha_test.go

```diff
@@ -130,15 +131,36 @@ func TestHA_waitForEnvironment(t *testing.T) {
 	wg.Add(1)
 
 	go func() {
-		ha.waitForEnvironment(chEnv)
-		assert.Equal(t, []byte("my.env"), ha.super.EnvId)
+		env := ha.waitForEnvironment(chEnv)
+		assert.Equal(t, []byte("my.env"), env.ID)
 		wg.Done()
 	}()
 
 	chEnv <- &Environment{ID: []byte("my.env")}
 	wg.Wait()
 }
 
+func TestHA_setAndInsertEnvironment(t *testing.T) {
+	ha := createTestingHA(t, testbackends.RedisTestAddr)
+
+	env := Environment{
+		ID: utils.EncodeChecksum(utils.Checksum("herp")),
+		Name: "herp",
+	}
+
+	err := ha.setAndInsertEnvironment(&env)
+	require.NoError(t, err, "setAndInsertEnvironment should not return an error")
+
+	rows, err := ha.super.Dbw.SqlFetchAll(mysqlTestObserver,
+		"SELECT name from environment where id = ? LIMIT 1",
+		ha.super.EnvId,
+	)
```

Comment:
IMAO a such function call's `(` `)` and the params have to be on separate lines.